terraform/redis/cluster/provision/data.tf

```diff
@@ -31,7 +31,7 @@ locals {
     256 = "cache.r5.12xlarge"
   }
 
-  node_type = length(var.node_type) == 0 ? local.instance_types[var.cache_size] : var.node_type
+  node_type = length(var.node_type) == 0 ? try(local.instance_types[var.cache_size], "") : var.node_type
```

Comment:
praise: so it works!! Great addition! :heart:

Reply:
Thanks for showing Terraform's `try` expressions to me! :)